src/kata5.reachDestination.js

```diff
@@ -1,3 +1,6 @@
-const reachDestination = (distance, speed) => {};
-
+const reachDestination = (distance, speed) => {
+  const roundHalf = (num) => Math.round(num * 2) / 2;
```

Comment:
I would refactor this to remove the `roundHalf` function, and instead just call Math.round... after calculating the time, as you're only calling it once e.g.
```
const time = distance / speed
const estimatedTime = Math.round(time * 2) / 2
```